plato/draw/fresnel/Ellipsoids.py

```diff
@@ -0,0 +1,43 @@
+import fresnel
+import itertools
+import numpy as np
+from ... import draw
+from ..internal import ShapeAttribute, ShapeDecorator
+from .FresnelPrimitive import FresnelPrimitive
+
+def fibonacciPositions(n_b, a=.5, b=0.5, c=0.5):
+    m = np.arange(n_b).astype(np.float32)
+    phi = m*np.pi*(3 - np.sqrt(5))
+    vy = 2*m/n_b + 1/n_b - 1
+    return np.array([a*np.sqrt(1 - vy**2)*np.cos(phi),
+                     b*vy, c*np.sqrt(1 - vy**2)*np.sin(phi)]).T
+
+
+@ShapeDecorator
+class Ellipsoids(draw.Ellipsoids, FresnelPrimitive):
+    __doc__ = draw.Ellipsoids.__doc__
+
+    _ATTRIBUTES = draw.Ellipsoids._ATTRIBUTES + list(
+        itertools.starmap(ShapeAttribute, [
+        ('outline', np.float32, 0, 0, False,
+         'Outline width for all particles'),
+        ('vertex_count', np.int32, 256, 0, False,
```

Comment:
pythreejs uses 64 vertices and it looks alright, but 64 looks pretty jagged in fresnel. I increased that to 256, which may make the rendering longer (I didn't time it). I'm fine with changing it to another value if desired.